packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -208,6 +211,10 @@ const onRenderParticipantDefault = (
       /* @conditional-compile-remove(PSTN-calls) */
       participantState={callingParticipant.state}
       ariaLabelledBy={participantAriaLabelledBy}
+      /* @conditional-compile-remove(raise-hand) */
+      showCoinBorder={callingParticipant?.raisedHand ? true : false}
```

Comment:
I don't think adding this showCoinBorder prop for ParticipantItem is necessary. I think you can assign onRenderAvatar callback prop (on line 207) to a variable like this or similar:
```
  const onRenderAvatarWithRaiseHand = callingParticipant?.raisedHand
    ? (userId?: string, options?: CustomAvatarOptions, defaultOnRender?: (props: CustomAvatarOptions) => JSX.Element) =>
        onRenderAvatar?.(userId, { ...options, styles: { border: callingPalette.raiseHandGold } }, defaultOnRender)
    : onRenderAvatar;
```

Reply:
Thanks, yes. This is a good solution